tests/usecase/test_db_config_options.py

```diff
@@ -0,0 +1,85 @@
+import os
+import unittest
+from database_config import get_database_config
+
+
+class TestDatabaseConfigOptions(unittest.TestCase):
+    def clean_env(self):
+        """
+        Setting different environment variables for test in the same process
+        can lead to flaky tests.
+        """
+        if "DATABASE_URL" in os.environ.keys():
+            del os.environ["DATABASE_URL"]
+
+        for key in filter(
+            lambda x: x.startswith("MXRUNTIME_Database"),
+            list(os.environ.keys()),
+        ):
+            del os.environ[key]
+
+    def test_no_setup(self):
+        self.clean_env()
+        with self.assertRaises(RuntimeError):
+            get_database_config()
+
+    def test_mx_runtime_db_config(self):
+        """
+        Test is MXRUNTIME variables are set up no database configuration is returned
+        based on DATABASE_URL or VCAP_SERVICES
+        """
+        self.clean_env()
+        os.environ["MXRUNTIME_DatabaseType"] = "PostgreSQL"
+        os.environ[
+            "MXRUNTIME_DatabaseJdbcUrl"
+        ] = "postgres://username:password@rdsbroker-testfree-nonprod-1-eu-west-1.asdbjasdg.eu-west-1.rds.amazonaws.com:5432/testdatabase"  # noqa E501
+
+        config = get_database_config()
+        assert not config
```

Comment:
This implies that the `config` value is falsy. Is this the intention? So it's okay if the config is `0` or `""`? If not, then please explicitly check for `None`

Reply:
Yes, for this test that is okay. we just want to make sure we get no config.